### PR TITLE
[14.0][ADD] cetmix_tower_server_yaml Yaml support for Cetmix Tower

### PR DESCRIPTION
**Pull request by ivs-cetmix**
This module implements YAML export/import for Cetmix Tower.

Following features are added in this PR:

- Save Command as yaml
- Populate Command values from yaml

Please check the module documentation for more details.

### CODE REVIEW

**Comment by ivs-cetmix** [comment]
Hi @norlinhenrik does this look like something you were asking for? Would appreciate your intermediate review in any case)

File: cetmix_tower_server_yaml/readme/USAGE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+To export data into YAML format:
+
+- Open the record you would like to export
+- Copy  YAML code from the text field located under the "YAML" tab in the form view
+
+To import data from YAML:
+
+- Open the record you would like to export
```
Comment (Aldeigja) [inline cetmix_tower_server_yaml/readme/USAGE.md]: `- Open the record you would like to import
`

**Review by GabbasovDinar — COMMENTED** [review]
Code LGTM!

**Review by GabbasovDinar — APPROVED** [review]

**Review by Aldeigja — CHANGES_REQUESTED** [review]

File: cetmix_tower_server_yaml/readme/USAGE.md (hunk 1 above)
Comment (ivs-cetmix) [reply cetmix_tower_server_yaml/readme/USAGE.md]: @Aldeigja fixed

**Review by GabbasovDinar — APPROVED** [review]

**Review by Aldeigja — APPROVED** [review]
Functional LGTM

**Comment by CetmixGitDrone** [comment]
Hey, thanks for contributing! Proceeding to merge this for you.
Prepared branch [14.0-dev-ocabot-merge-pr-103-by-ivs-cetmix-bump-nobump](https://github.com/cetmix/cetmix-tower/commits/14.0-dev-ocabot-merge-pr-103-by-ivs-cetmix-bump-nobump), awaiting test results.

**Comment by CetmixGitDrone** [comment]
Congratulations, your PR was merged at 4bf482c65c63fcd97417a08bfac6ad6ab4b8a317. Thanks a lot for contributing to cetmix. ❤️

**Pull request merged**